.github/workflows/close.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow closes issues that have had no activity for 90 days.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Close Old
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 1 1,15 * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 30
+        close-issue-message: >
```

Comment:
Copy across feedback from #3047 

I think the better way to do this is to use `close-issue-message` and `days-before=close` on the obsolete workflow, since that is the workflow I see in the github action docs:

e.g. 
https://github.com/actions/stale/tree/main#close-issue-message:~:text=Configure%20different%20stale%20timeouts%3A

It seems like we could do this in two yaml files, rather than three.

Reply:
We talked about this offline, ignore this concern.